### PR TITLE
docs(blogpost): poll joining blog

### PR DESCRIPTION
**Pull request by djanluka**
# Description

PR with a blogpost of the Anonymous poll joining grant.

## Related issue(s)

The blogpost for #1566 

## Confirmation

- [x] I have read and understand MACI's [contributor guidelines](https://maci.pse.dev/docs/contributing) and [code of conduct](https://maci.pse.dev/docs/contributing/code-of-conduct).
- [x] I have read and understand MACI's [GitHub processes](https://github.com/privacy-scaling-explorations/maci/discussions/847).
- [x] I have read and understand MACI's [testing guide](https://maci.pse.dev/docs/testing).

### CODE REVIEW

**Review by ctrlc03 — COMMENTED** [review]
Seems like there's some build error, try inside `apps/website` to run `pnpm build` please

File: apps/website/blog/2024-08-29-poll-joining.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+### What is poll joining?
+
+Poll joining represents a protocol change where each poll contains its own poll state tree that stores the keys of users who want to vote on that poll. The users join the poll by providing a new key that they will use in that specific poll. But wait, then anyone can create as many keys as they like and vote many times so we lose everything? Not really. The users submit their new public key and ZK proof that they own some key in the original MACI state tree. That way, only registered users can join the polls without revealing the link between the old and the new key. To prevent multiple joins, the user also provides a nullifier - computed as a hash of their private key. The submitted ZK proof also verifies that the nullifier is correctly computer from the same private key associated with the public key in the original MACI state tree. On the smart contract side, the user is prevented from joining the poll with an already spent nullifier.
```
Comment (ctrlc03) [inline apps/website/blog/2024-08-29-poll-joining.md:21]: typo: "correctly computer" -> "correctly computed"

**Review by ctrlc03 — CHANGES_REQUESTED** [review]
LGTM, just one typo

File: apps/website/blog/2024-08-29-poll-joining.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+slug: poll-joining-improves-anonymity
+title: Poll joining improves anonymity - but wait, there is more!
```
[the remark this answers is not in the document]
Comment (djanluka) [reply apps/website/blog/2024-08-29-poll-joining.md:3]: It sounds me better too, I'll update

[the remark this answers is not in the document]
Comment (djanluka) [reply apps/website/blog/2024-08-29-poll-joining.md]: https://github.com/privacy-scaling-explorations/maci/pull/1798/commits/84a45bf5802cec69bdbc9b8da32624307d8f992c

File: apps/website/blog/2024-08-29-poll-joining.md (hunk 1 above)
Comment (djanluka) [reply apps/website/blog/2024-08-29-poll-joining.md]: https://github.com/privacy-scaling-explorations/maci/pull/1798/commits/6d27f6fe0a9eb48a3d7bed060a535877ee199baf

**Review by ctrlc03 — APPROVED** [review]
🚀

**Comment by ctrlc03** [comment]
@djanluka not sure what happened but I cannot rebase it from the ui, could you please rebase this branch to dev so it will let me merge it?

**Comment by ctrlc03** [comment]
now there's 11 commits, please use rebase 

Screenshot 2024-09-02 at 10 59 42

**Comment by djanluka** [comment]
I pulled `dev` into `blog/poll-joining`, but had `pull.rebase false` in the git config.. Could you merge with 11 commits?

**Comment by ctrlc03** [comment]
> I pulled dev into blog/poll-joining, but had pull.rebase false in the git config.. Could you merge with 11 commits? @ctrlc03

@djanluka So we don't use merge, we use rebase in this repo. If you see, now this PR is trying to merge files that are already part of the codebase and have nothing to do with the blogpost. Please cleanup the PR and ensure that only the relevant commits and files are part of it.

**Review by kittybest — APPROVED** [review]
thanks!

Meanwhile, I suggest squash all the commits in this PR before you rebase, that would make your day better.

**Comment by djanluka** [comment]
I reverted unnecessary changes, but you still can't rebase?

**Comment by ctrlc03** [comment]
> I reverted unnecessary changes, but you still can't rebase?

please run 

```
git reset $(git merge-base dev $(git rev-parse --abbrev-ref HEAD))
```

and force push so there is only one commit and a clean history

**Comment by djanluka** [comment]
> (git rev-parse --abbrev-ref HEAD

done, but again there are 7 files changed

**Comment by djanluka** [comment]
if doesn't work for you, we might raise a new clear pull request?

**Pull request closed without merging**